src/container/srv_container.c

```diff
@@ -30,6 +30,7 @@
 
 #define DAOS_POOL_GLOBAL_VERSION_WITH_CONT_MDTIMES 2
 #define DAOS_POOL_GLOBAL_VERSION_WITH_CONT_NHANDLES 2
+#define DAOS_POOL_GLOBAL_VERSION_WITH_CONT_EX_EVICT 2
```

Comment:
Ah, I think you are right to manage this with the layout version. I failed to realize that this actually is related to layout (new flag bits are stored in the pool handle). Yes, I think CONT_PROTO_VER applies to all resources (it is associated with the software version as a whole). Yeah, I think I have heard on the matter of upgrading pools to the new layout that you need to be really sure because it is a one-way operation. It would follow also that if you are going to do such layout upgrades, as a daos admin, you would need to be sure to complete that upgrade for all pools in the system.